Add radial integral for custom densities (#298)
Luthaf/rascaline: docs/src/how-to/long-range.rst

.. _userdoc-how-to-long-range-descriptor:

> Pure LODE long-range descriptor

Reviewer: I'm not sure if "pure LODE" is the best way to call this descriptor. Maybe "long-range only LODE descriptor"?

Author: Yes, I agree.

===============================

The :py:class:`rascaline.LodeSphericalExpansion` allows the calculation of a descriptor that
includes all atoms within the system and projects them onto a spherical expansion/
fingerprint within a given ``cutoff``. This is very useful if long-range interactions
between atoms are important to describe the physics and chemistry of a collection of
atoms. However, as stated the descriptor contains **ALL** atoms of the system and
sometimes it can be desired to only have a pure long-range/exterior descriptor that only
includes the atoms outside a given cutoff.

.. figure:: ../../static/images/long-range-descriptor.*
    :align: center
    :class: only-light

.. figure:: ../../static/images/long-range-descriptor-dark.*
    :align: center
    :class: only-dark

This descriptor can be particular useful when already having a good descriptor for the

> short-range density like (SOAP) and the long-range descriptor (far field) should not
> have the information the short-range descriptor already contains. 

Reviewer: ```suggestion
short-range density like (SOAP) and the long-range descriptor (far field) should contain
different information from what the short-range descriptor already offers. 
```
